cognite_toolkit/_cdf_tk/load/_resource_loaders.py

```diff
@@ -2132,6 +2263,31 @@ def drop_data(self, ids: SequenceNotStr[NodeId]) -> int:
         # Nodes will be deleted in .delete call.
         return 0
 
+    @classmethod
+    @lru_cache(maxsize=1)
+    def get_write_cls_parameter_spec(cls) -> ParameterSpecSet:
+        spec = super().get_write_cls_parameter_spec()
+        # Modifications to match the spec
+        for item in spec:
+            if item.path[0] == "apiCall" and len(item.path) > 1:
+                # Move up one level
+                # The spec class is immutable, so we use this trick to modify it.
```

Comment:
You can use `dataclasses.replace` I believe, unless you actually need to mutate them directly

Reply:
I do want to mutate them directly.